cases/sider/Sider-Config.md

```diff
@@ -0,0 +1,28 @@
+# 这是一个指导你如何在Sider程序中应用本程序的API
+Sider ，一个浏览器插件
+
+https://chrome.google.com/webstore/detail/difoiogjjojoaoomphldepapgpbgkhkb
```

Comment:
Is this open source?

Reply:
I am no sure. 
It is a guide to USE ChatGPT-to-API to Crack Any OPENAI API KEY Setting.
I have nothing to do with them, I just do a show to be used on other platforms